packages/core/src/device/DevicePool.ts

```
@@ -247,4 +247,17 @@ export class DevicePool extends EventEmitter {
 
     this.disconnectPool.push(descriptor);
   }
+
+  static resetState() {
+    this.current = null;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)